integration-test/src/test/java/uk/gov/gchq/gaffer/integration/impl/loader/AddElementsLoaderIT.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.integration.impl.loader;
+
+import uk.gov.gchq.gaffer.data.element.Element;
+import uk.gov.gchq.gaffer.operation.impl.add.AddElements;
+
+public class AddElementsLoaderIT extends AbstractLoaderIT<AddElements> {
+
+    @Override
+    public void configure(final Iterable<? extends Element> elements) {
+        // Empty
+    }
+
+    @Override
+    public AddElements createOperation(final Iterable<? extends Element> elements) {
```

Comment:
I think we need to have some other tests to check failures and check what happens with the different flags are set: validate and skipInvalidElements.